test/ts/preact.tsx

```diff
@@ -29,7 +35,7 @@ function DummerComponent({ input, initialInput }: DummerComponentProps) {
 	return <div>Input: {input}, initial: {initialInput}</div>;
 }
 
-render(h(DummerComponent, { initialInput: "The input", input: "New input" }), document);
+render(h(DummerComponent, { initialInput: "The input", input: "New input", key: "1", jsx: false }), document);
```

Comment:
A quick search doesn't turn up any mention of a special `jsx` attribute. Seems like this is an error in our typings? /cc @developit

Reply:
Hmm agreed. It seems it was added in #737. @valotas, do you know what the `jsx` prop defined in the `Attributes` interface is meant to do? Doesn't appear Preact does anything with it. Does TypeScript use it for something?

Reply:
@matthewmueller [suggested](https://github.com/developit/preact/pull/737#issuecomment-328071941) the `jsx` attribute. I personally never used it, but it looks like some frameworks do use this. It does nothing as fas as I understood but having it defined in the types helps making use of it (when needed). Of course you can always define it in your props.

Since #737 has not been published yet, we could remove it if @developit and/or @marvinhagemeister does not like it.